neutron/templates/bin/_neutron-openvswitch-agent.sh.tpl

```
@@ -2,6 +2,16 @@
 set -x
 chown neutron: /run/openvswitch/db.sock
 
+# ensure we can talk to openvswitch or bail early
+# this is until we can setup a proper dependency
+# on deaemonsets - note that a show is not sufficient
+# here, we need to communicate with both the db and vswitchd
+# which means we need to do a create action
+# 
+# see https://github.com/att-comdev/openstack-helm/issues/88
+timeout 3m neutron-sanity-check --config-file /etc/neutron/neutron.conf --ovsdb_native --nokeepalived_ipv6_support
```

It is quite handy, even with the dependency stuff in place it makes sense at some level to take advantage of it as it can catch errors early and exit.  The neutron agent tends to mask errors such as failures working with br-int and just hang or skip foundational setup which is a terrible thing to do because it 'looks' like its working when it will never properly pass packets.